packages/devextreme/js/ui/pager.d.ts

```diff
@@ -0,0 +1,61 @@
+import Widget, { WidgetOptions } from './widget/ui.widget';
+import {
+    BasePagerOptions,
+    PagerAllPagesMode,
+} from '../common';
+
+/**
```

Comment:
Add re-export for public types:
After that run the regenerate script, these types should be added to the pager_types file.

```suggestion
import {
    BasePagerOptions,
    PagerAllPagesMode,
} from '../common';


export {
    PagerAllPagesMode,
} from '../common';

/**
```